src/widgets/principles/ui/principle-card/principles.tsx

```diff
@@ -8,7 +8,7 @@ export const Principles = () => (
   <div className="principles container">
     <div className="principles content">
       <WidgetTitle size="large" mods="lines">RS School Principles are an ability to complete our mission</WidgetTitle>
-      <div className="cards column-3 ">
+      <div className="column-3 cards">
```

Comment:
As i see in figma design, cards should go in one column on tablets and lower (810px max), but right now it's from 1400px (laptop). Also shouldnt we move styles to modules as was agreed before? It would definitly help with prev issue, cuz as i see it goes from that column-3 global class.

Reply:
We have been discussing sizes and margins with @IlliaSakharau in DM, and as far as I understand, the new figma layout mostly shows only what the content of the card should look like, but other staff only for reference and contex. Behaivor of other things should persist. Also, @IlliaSakharau approved this version (I messaged link to deploy), but I'm not entirely sure if he checked it on every screen size.

I don't think this is big deal, so if @IlliaSakharau approves, "cards should go in one column on tablets and lower (810px max)", sure I will change.

Reply:
>Also shouldnt we move styles to modules as was agreed before? It would definitly help with prev issue, cuz as i see it goes from that column-3 global class.

I suppose that 'move styles to modules' (.scss to module.scss) little out of scope bug fixing

Reply:
Actually, I suppose the behavior of current breakpoints should persist due to display problems three col cards on smaller sizes:
![image](https://github.com/user-attachments/assets/cc1d44c8-d072-418a-a1cf-3a8a063f8533)
![image](https://github.com/user-attachments/assets/8131fcd9-9a1d-4003-be21-ce04df66a575)

Reply:
> I suppose that 'move styles to modules' (.scss to module.scss) little out of scope bug fixing
i do not insist, just as i got it there was an agreement to make this transition to modules if we make significant changes in file.


> Actually, I suppose the behavior of current breakpoints should persist due to display problems three col cards on smaller sizes:
i see. but on 1400px in column it looks not much better) let's see what @Illia-Sakharau will say.

Reply:
@Wystov  @thirdmadman 
It is absolutely true that you should not stick rigidly to Figma mocks.
In my opinion, this implementation looks good.
We can change the breakpoint of 1440px to something like 1168px, but I don't think it's necessary.